src/Resources/views/partials/inner.php

```diff
@@ -1,4 +1,7 @@
+<link rel="stylesheet" href="<?=$this->uri() . '/pub/' . $this->asset('css/main.css')?>" />
```

Comment:
Unfortunately yes :disappointed: 

Everything in the head section is ignored when using `jquery.load` as done in https://github.com/owncloud/core/pull/22255. If you have a cleaner solution I'm happy for suggestions :smile: